casanovo/denovo/model.py

```diff
@@ -12,6 +12,7 @@
 from depthcharge.models.embed.model import PairedSpectrumEncoder
```

Comment:
**question (non-blocking, irrelevant):** Nothing to do with this PR, but do you know why this import exists here?

Reply:
It has always been allowed as an option for the encoder of a `Spec2Pep` object, although we always just use a `SpectrumEncoder` for Casanovo. Maybe a dangling copy/paste issue?

Reply:
I think so. We can delete in another PR.